lib/puppet/type/cisco_vni.rb

```diff
@@ -0,0 +1,85 @@
+# Manages a Cisco VNI.
```

Comment:
*nit* Let's change this to..

`#Manages a Cisco Virtual Network Interface (VNI)`  or whatever VNI stands for :)